services/viva-ms/lambdas/generateRecurringCaseHtml.js

```diff
@@ -29,6 +29,36 @@ handlebars.registerHelper({
   },
 });
 
+async function getClosedUserCases(primaryKey) {
+  const dynamoDbQueryCasesParams = {
+    TableName: config.cases.tableName,
+    KeyConditionExpression: 'PK = :pk and begins_with(SK, :sk)',
+    FilterExpression: 'begins_with(#status.#type, :statusTypeClosed)',
+    ExpressionAttributeNames: {
+      '#status': 'status',
+      '#type': 'type',
+    },
+    ExpressionAttributeValues: {
+      ':pk': primaryKey,
```

Comment:
I who am picky but a primary key is built up by both PK and SK (sort key). In this case, you only enter a PK = partition key.
Feel free to rename the function argument to `pk` or better to `partitionKey` :)

Reply:
Done